docs/pages/Builder.md

```diff
@@ -48,38 +48,32 @@ A project can add custom tasks to the build by using the [Custom Tasks Extensibi
 
 ### Standard Tasks
 
-All available standard tasks are documented [in the API reference](https://sap.github.io/ui5-tooling/api/module-@ui5_builder.tasks.html) and are listed below in the order of their execution:
-
-| Task | Type `application` | Type `library` | Type `theme-library` |
-| ---- | :----: | :----: | :----: |
-| escapeNonAsciiCharacters | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } |  |
-| replaceCopyright | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } |
-| replaceVersion | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } |
-| replaceBuildtime |  | {: .sap-icon-circle-task-2 } |  |
-| generateJsdoc |  | {: .sap-icon-circle-task }^1^ |  |
-| executeJsdocSdkTransformation |  | {: .sap-icon-circle-task }^1^ |  |
-| generateFlexChangesBundle |  | {: .sap-icon-circle-task-2 } |  |
-| generateManifestBundle | {: .sap-icon-circle-task } | {: .sap-icon-circle-task } |  |
-| generateLibraryManifest |  | {: .sap-icon-circle-task-2 } |  |
-| generateLibraryPreload |  | {: .sap-icon-circle-task-2 } |  |
-| generateComponentPreload | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task }^2^ |  |
-| generateStandaloneAppBundle | {: .sap-icon-circle-task }^3^ |  |  |
-| transformBootstrapHtml | {: .sap-icon-circle-task }^3^ |  |  |
-| generateBundle | {: .sap-icon-circle-task }^4^ | {: .sap-icon-circle-task }^4^ |  |
-| buildThemes |  | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } |
-| generateThemeDesignerResources |  | {: .sap-icon-circle-task } | {: .sap-icon-circle-task } |
-| createDebugFiles | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } |  |
-| uglify | {: .sap-icon-circle-task-2 } | {: .sap-icon-circle-task-2 } |  |
-| generateVersionInfo | {: .sap-icon-circle-task-2 } |  |  |
-| generateCachebusterInfo | {: .sap-icon-circle-task } |  |  |
-| generateApiIndex | {: .sap-icon-circle-task }^1^ |  |  |
-| generateResourcesJson | {: .sap-icon-circle-task } | {: .sap-icon-circle-task } | {: .sap-icon-circle-task } |
-
-*Enabled by default*
-{: .sap-icon-circle-task-2-before }
-
-*Disabled by default. Can be activated by certain build modes, project configuration or by using the `--include-task` [CLI parameter](./CLI.md#ui5-build). See footnotes where given* 
-{: .sap-icon-circle-task-before }
+All available standard tasks are documented [in the API reference](../../api/index.html). Search for `@ui5/builder/tasks/` to filter the API reference for all available tasks. The list below offers the actual order of their execution:
```

Comment:
The problem with the API reference is that it's not available when *serving* the documentation locally (only when building). AFAIK that's the reason we used absolute links to GitHub pages in the past. So not sure whether we want to change this.

Maybe we could come up with a script as part of the documentation build that replaces `https://sap.github.io/ui5-tooling/` with a relative URL 🤔 Might be tricky since the amount of "../" depends on the page configured in the mkdocs.yml (not the physical location of the file)